db/seed/user.rb

```diff
@@ -0,0 +1,14 @@
+p 'Start seeding Users'
+first_names = %w[Mike John Kelly]
+last_names = %w[Dow Jonson Jordan]
+
+if User.admin.count.zero?
```

Comment:
Please, create just one admin user. and use find_or_create_by instead of count check